conformance/conf_tests/__init__.py

```diff
@@ -38,43 +38,43 @@
                                test_y_smoothscale, test_varied_smoothscale,
                                test_subsurface_smoothscale)
 
-conformance_tests = {
-    'test_rgba_add': test_rgba_add,
-    'test_rgba_sub': test_rgba_sub,
-    'test_rgba_min': test_rgba_min,
-    'test_rgba_max': test_rgba_max,
-    'test_rgba_mult': test_rgba_mult,
-    'test_rgb_mult': test_rgb_mult,
-    'test_blend_mult': test_blend_mult,
-    'horz1': test_horz_line,
-    'horz_widths': test_horz_line_width,
-    'vert1': test_vert_line,
-    'vert_widths': test_vert_line_width,
-    'lines': test_lines,
-    'lines_width': test_lines_width,
-    'lines_func': test_lines_function,
-    'scale': test_scale,
-    'flip': test_flip,
-    'flip_subsurface': test_flip_subsurface,
-    'rotate': test_rotate,
-    'rect': test_rect,
-    'polygon': test_polygon,
-    'scroll': test_scroll,
-    'smooth_int': test_int_smoothscale,
-    'smooth_x': test_x_smoothscale,
-    'smooth_y': test_y_smoothscale,
-    'smooth_varies': test_varied_smoothscale,
-    'smooth_subsurface': test_subsurface_smoothscale,
-    'chop': test_chop,
-    'chop_subsurface': test_chop_subsurface,
-    'rotozoom': test_rotozoom,
-    'hollow_circles': test_hollow_circles,
-    'filled_circles': test_filled_circles,
-    'filled_ellipses_1': test_filled_ellipses_1,
-    'filled_ellipses_2': test_filled_ellipses_2,
-    'hollow_ellipses': test_hollow_ellipses,
-    'limit_circles': test_filled_circles_limits,
-}
+conformance_tests = [
+        test_horz_line,
+        test_horz_line_width,
+        test_vert_line,
+        test_vert_line_width,
+        test_lines,
+        test_lines_width,
+        test_lines_function,
+        test_subsurface_smoothscale,
+        test_varied_smoothscale,
+        test_y_smoothscale,
+        test_x_smoothscale,
+        test_int_smoothscale,
+        test_filled_circles_limits,
+        test_hollow_ellipses,
+        test_filled_ellipses_1,
+        test_filled_ellipses_2,
+        test_filled_circles,
+        test_hollow_circles,
+        test_rect,
+        test_polygon,
+        test_scroll,
+        test_rgba_add,
+        test_rgba_sub,
+        test_rgba_min,
+        test_rgba_max,
+        test_rgba_mult,
+        test_rgb_mult,
+        test_blend_mult,
+        test_rotozoom,
+        test_scale,
+        test_flip,
+        test_flip_subsurface,
+        test_rotate,
+        test_chop,
+        test_chop_subsurface,
+]
```

Comment:
We could, but when working on debugging a conformance test issue, it is very useful to just restrict runs to the failing tests, so we do need to add some way of restricting the tests run if we go that route.